services/CollectionService/src/index.php

```diff
@@ -22,23 +23,12 @@
   'twig.path' => __DIR__.'/../templates',
 ));
 
-$islandoraResourceServiceProvider = new \Islandora\ResourceService\Provider\ResourceServiceProvider;
```

Comment:
It's totally non-functional.  Was just trying to make a point about injecting things in and making sure to use PHP's type system for gurantees.  I've yet to pull out the rest of the Resource Service stuff.